crates/bevy_render/Cargo.toml

```diff
@@ -57,9 +57,12 @@ bevy_tasks = { path = "../bevy_tasks", version = "0.12.0-dev" }
 image = { version = "0.24", default-features = false }
 
 # misc
-wgpu = { version = "0.16.0", features=["naga"] }
 codespan-reporting = "0.11.0"
-naga = { version = "0.12.0", features = ["wgsl-in"] }
+# `fragile-send-sync-non-atomic-wasm` feature means we can't use WASM threads for rendering
+# It is enabled for now to avoid having to do a significant overhaul of the renderer just for wasm
+wgpu = { version = "0.17.1", features = ["naga", "fragile-send-sync-non-atomic-wasm"] }
```

Comment:
Should this use 17.2 per @cwfitzgerald comment?

Reply:
0.17.2 was `wgpu-hal` only, there's no 0.17.2 for `wgpu` (and cargo will automatically choose the highest patch version here).

Reply:
Ah, my mistake, cheers